### ISSUE TITLE
Abandoned and Archived Groups keep appearing in Main Chats list

### ISSUE THREAD

**Issue by lorthirk** [telegramdesktop/tdesktop#16406]
### Steps to reproduce
1. Join a group
2. Abandon said group
3. Archive the chat
3. Close Telegram
4. Reopen Telegram

### Expected behaviour
Once abandoned, the group should never appear again even if the chat is not deleted but only archived

### Actual behaviour
The group chat is shown in the main list even after archiving it

### Configuration
**Operating system:** Windows 10

**Version of Telegram Desktop:** 2.7.4

**Used theme**: Night

**Comment by RememberTheAir** [comment]
Wait how do you leave a group without completely deleting it from your chats list? To archive a chat, you must see that chat in the chats list

**Comment by lorthirk** [comment]
Leaving a group chat doesn't mean you delete its history.

**Comment by RememberTheAir** [comment]
I only see an option to "delete and leave" from tdesktop. You're leaving the chat from another app that allows to retain its history maybe?

**Comment by lorthirk** [comment]
Well, at least in the Mac version (but I guess it's different?) there's an option to just Leave the group:

![image](https://user-images.githubusercontent.com/8140139/120990432-d5b3be00-c780-11eb-9d84-5ea595fc0dc6.png)
 
Will update with the Windows version shortly

**Comment by jonbesga** [comment]
Happening to me as well. I have around 15 "dead" groups in the list

**Comment by RememberTheAir** [comment]
Telegram Desktop's "leave group" option should delete the group from your chats list. You're probably seeing those groups in your chats list because:
- those groups were not supergroups
- you have been removed by an admin from them

Both of these conditions must be true for a dead group to appear in your chats list. Might it be the case @jonbesga @lorthirk? If so, it's the intended behavior. If not, the it's a bug

**Comment by lorthirk** [comment]
No, I don't think it's my case. For sure I left them myself, I wasn't removed by the admin.

**Comment by lorthirk** [comment]
The issue is still present, and by the way Innoticed that more abandoned groups are also shown in Telegram WebZ.

**Comment by Aokromes** [comment]
maybe api bug then?

**Comment by lorthirk** [comment]
I definitely NOT lost interest on this item, since it's still present as of today.

**Comment by Zensonaton** [comment]
Same problem for me!

**Comment by ntzolov** [comment]
I have this problem/bug too. I cant get rid out of a dead group...

**Comment by lorthirk** [comment]
Again, the issue still peesist

**Comment by lorthirk** [comment]
Still there.

**Comment by lorthirk** [comment]
Still happening

**Comment by lorthirk** [comment]
I'm nor giving up :)

**Comment by Aokromes** [comment]
> The issue is still present, and by the way I noticed that more abandoned groups are also shown in Telegram WebZ.

y quote myself 

> maybe api bug then?
